tests/lib/rules/arrow-parens.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @fileoverview Tests for arrow-parens
+ * @author Jxck
+ * @copyright 2015 Jxck. All rights reserved.
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+var eslintTester = new ESLintTester(eslint);
+
+var valid = [
+    "() => {}",
+    "(a) => {}",
+    "(a) => a",
+    "(a) => {\n}",
+    "a.then((foo) => {});",
+    "a.then((foo) => { if (true) {}; });"
+].map(function(code) {
```

Comment:
Can you break these out so they don't use `map`? We try to keep tests all looking fairly consistent so they're easier to work with.